VWOSdk/Core/VWOClient.cs

```diff
@@ -624,6 +627,71 @@ public bool FlushEvents()
                     LogInfoMessage.NoDataUserStorageService(file, campaignKey, userId);
                 }
             }
+
+            // Mutually Exclusive Group
+            IDictionary<string, dynamic> groupDetails = CampaignHelper.isPartOfGroup(this._settings, campaign.Id);
+            if (groupDetails.ContainsKey("groupId"))

```

Comment:
where is percent traffic check for called campaign at the beginning if false it should return from there only